docs/s3.md

```diff
@@ -0,0 +1,75 @@
+# Performance Test Result storage in S3 bucket for SLV
+
+**NOTE: This document is only relevant to puppet employees**
+
+The SLV team has an s3 bucket for storing results from performance testing.
+
+`
+s3://slv-performance-results
+`
+
+## Convention
+
+S3 is [object storage](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/using-folders.html),
+so it doesn't have true directories.  But it does like to make it look like it does.  The convention 
```

Comment:
Delete the "But it does..." incomplete sentence or make it a clause to the previous sentence.